### ISSUE TITLE
latest firebase_messaging plugin broken with android on flutter 1.7

### ISSUE THREAD

**Issue by johnhouston00** [mwaylabs/flutter-apns#6]
I believe this is related to this issue, https://github.com/FirebaseExtended/flutterfire/issues/188 just recently( the 19th ), firebase_messaging was upgraded, however, on android using Flutter 1.7 ( not the latest upgrade 1.9 ).  I receive this error using flutter-apns 1.0.2.

```
error: method findAppBundlePath in class FlutterMain cannot be applied to given types;
String appBundlePath = FlutterMain.findAppBundlePath();
^
required: Context
found: no arguments
reason: actual and formal argument lists differ in length
1 error
Finished with error: Gradle task assembleDebug failed with exit code 1
```
Reverting flutter-apns to 1.0.1 seems to work for android, but does not seem to compile properly on iOS.

We are not ready to update to flutter 1.9( if this is truly the issue ), changing the firebase_messaging include in flutter-apns pubspec.yaml to be "5.1.3" rather than "^5.1.3" seems to fix this issue when I build flutter-apns as a local plugin.

**Comment by szotp** [comment]
I will check if 5.1.3 works with 1.9. If yes I can pin to 5.1.3 or 5.1.5 until new firebase comes out.

**Comment by szotp** [comment]
Fixed this in version 1.0.3.

**Comment by szotp** [comment]
I'm assuming problem was fixed, otherwise please reopen.
